### PR TITLE
CI: Update CEF version to 4638 (Chromium 95) on all platforms

### PR DESCRIPTION
**Pull request by WizardCM**
### Description

This updates the obs-browser dependency CEF to Chrome 95, released November 2021 on all platforms, including Flatpak.

Known issues (likely to be race conditions):

1. Linux: a (random) shutdown crash when a browser source **or** panel is open
2. Linux: a (reproducible) shutdown crash when a browser source is open (verifiable with Twitch specifically, but not the default url)
3. Linux: a (reproducible) crash fixed by https://github.com/obsproject/obs-browser/commit/f7431948d24f502f2185fa03e1bd55350eef7107

The macOS & Windows builds include shared texture support, and additionally all platforms include a new API (currently unused) to provide per-source permission control for microphones, webcams & display capture.

### Motivation and Context

Outdated CEF is bad for various reasons. Users also ask for features unavailable in the current versions provided.

This update additionally fixes a stutter/crackle that users occasionally hear through the "Control audio via OBS" checkbox.

### How Has This Been Tested?

Built on CI, launched on Windows, macOS & Linux (via Flatpak).

### Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Performance enhancement (non-breaking change which improves efficiency)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -275,7 +275,7 @@ jobs:
 
           mv ./libobs-opengl/libobs-opengl.so ./OBS.app/Contents/Frameworks
 
-          cp -R "${{ github.workspace }}/cmbuild/cef_binary_${{ env.MACOS_CEF_BUILD_VERSION }}_macosx64/Release/Chromium Embedded Framework.framework" ./OBS.app/Contents/Frameworks/
+          cp -R "${{ github.workspace }}/cmbuild/cef_binary_${{ env.MACOS_CEF_BUILD_VERSION }}_macos_x86_64/Release/Chromium Embedded Framework.framework" ./OBS.app/Contents/Frameworks/
```
Comment (gxalpha) [inline .github/workflows/main.yml:278]: These paths also need to be changed in `CI/full-build-macos.sh`

Comment (WizardCM) [reply .github/workflows/main.yml:278]: Good catch! I did check for version numbers, but forgot to check filenames.

**Review by GeorgesStavracas — APPROVED** [review]
Thanks for updating the Flatpak file together with the Linux job. Known crashes aside, I think the Linux changes of this PR look good.

**Comment by Julusian** [comment]
Where can I find the sources/patch used to build the 4638 cef binaries used here?
I am wanting to use these binaries in another project which wants the shared_texture stuff (to save having to build it myself), but need to have the source just in case

**Pull request merged**